Source/CarthageKit/GitHub.swift

```diff
@@ -306,8 +304,10 @@ extension GitHubRelease: CustomStringConvertible {
 }
 
 extension GitHubRelease: Decodable {
-	public static func create(ID: Int)(name: String?)(tag: String)(draft: Bool)(prerelease: Bool)(assets: [Asset]) -> GitHubRelease {
-		return self.init(ID: ID, name: name, tag: tag, draft: draft, prerelease: prerelease, assets: assets)
+	public static func create(ID: Int) -> String? -> String -> Bool -> Bool -> [Asset] -> GitHubRelease {
+		return { name in { tag in { draft in { prerelease in { assets in
+			return self.init(ID: ID, name: name, tag: tag, draft: draft, prerelease: prerelease, assets: assets)
+		} } } } }
```

Comment:
I know that framework, but Curry also uses curried function syntax currently, so it's not the solution here.